web/src/features/hfiCalculator/components/CalculatedPlanningAreaCells.tsx

```diff
@@ -4,15 +4,16 @@ import { StationDaily } from 'api/hfiCalculatorAPI'
 import FireStartsCell from 'features/hfiCalculator/components/FireStartsCell'
 import {
   calculateDailyMeanIntensities,
-  calculateMaxMeanIntensityGroup,
-  calculateMeanIntensityGroupLevel
+  calculateMaxMeanIntensityGroup
 } from 'features/hfiCalculator/components/meanIntensity'
 import MeanIntensityGroupRollup from 'features/hfiCalculator/components/MeanIntensityGroupRollup'
 import PrepLevelCell from 'features/hfiCalculator/components/PrepLevelCell'
 import { NUM_WEEK_DAYS } from 'features/hfiCalculator/constants'
 import { getDailiesForArea } from 'features/hfiCalculator/util'
 import { groupBy, range } from 'lodash'
 import React from 'react'
+import AveragePrepLevelCell from './AveragePrepLevelCell'
+import { calculateDailyPrepLevels, calculateMeanPrepLevel } from './prepLevel'
```

Comment:
would prefer to see these as absolute filepaths for import, rather than relative

Reply:
@andrea-williams mind looking into adding absolute imports as a default setting in: https://github.com/bcgov/wps/blob/main/.vscode/settings.json

This comes up a lot and we spend too much time on something that can be automated

Reply:
Ah yes I will correct that